src/uTensor/ops/Convolution_kernels.hpp

```diff
@@ -1,18 +1,26 @@
 #ifndef UTENSOR_CONVOLUTION_KERNELS_H
 #define UTENSOR_CONVOLUTION_KERNELS_H
 #include <algorithm>
-#include <limits>
 
 #include "uTensor/core/operatorBase.hpp"
 
 namespace uTensor {
 
 enum Padding : uint8_t { UNKNOWN = 0, VALID = 1, SAME = 2 };
 
+// Matches TF's behavior, ref:
+// https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/framework/kernel_shape_util.cc#L23
```

Comment:
change the link to permanent link:
https://github.com/tensorflow/tensorflow/blob/6ea11589562f64a404ab74b1df9eba33f1fad83f/tensorflow/core/framework/kernel_shape_util.cc#L23